src/server.h

```diff
@@ -2413,6 +2415,8 @@ typedef int serverGetKeysProc(struct serverCommand *cmd, robj **argv, int argc,
  * CMD_TOUCHES_ARBITRARY_KEYS: The command may touch (and cause lazy-expire)
  *                             arbitrary key (i.e not provided in argv)
  *
+ * CMD_CROSS_SLOT: The command may access keys in different slots.
```

Comment:
We would need to expose this into the module API presumably, since there is a JSON command for multi-gets that can fan out. `JSON.MGET`.